app/src/main/java/org/p2p/wallet/solend/ui/earn/SolendEarnPresenter.kt

```diff
@@ -0,0 +1,47 @@
+package org.p2p.wallet.solend.ui.earn
+
+import android.content.Context
+import org.p2p.wallet.common.mvp.BasePresenter
+import org.p2p.wallet.solend.model.SolendDepositToken
+import org.p2p.wallet.utils.getErrorMessage
+import timber.log.Timber
+import java.math.BigDecimal
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+
+class SolendEarnPresenter(
+    private val context: Context
+) : BasePresenter<SolendEarnContract.View>(), SolendEarnContract.Presenter {
+
+    override fun load() {
```

Comment:
you can use onViewAttach

Reply:
I would propose not using attach for it. 
If we'll call presenter.load in fragment that would be more obvious and the developer should not enter the presenter to investigate what's being called in attach